hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/s3/security/TestS3GetSecretRequest.java

```diff
@@ -282,7 +283,7 @@ public void testFetchSecretForRevokedUser() throws IOException {
         ozoneManager, 2);
 
     // Verify that the revoke operation was successful.
-    assertTrue(omRevokeResponse instanceof S3RevokeSecretResponse);
+    assertInstanceOf(S3RevokeSecretResponse.class, omRevokeResponse);
     S3RevokeSecretResponse s3RevokeSecretResponse =
         (S3RevokeSecretResponse) omRevokeResponse;
```

Comment:
nit: cast can be removed

```suggestion
    S3RevokeSecretResponse s3RevokeSecretResponse =
        assertInstanceOf(S3RevokeSecretResponse.class, omRevokeResponse);
```

(Also in other test cases in this class.)